languages/clojure/exercises/concept/lists/.meta/example.clj

```diff
@@ -20,7 +20,7 @@
   (count lang-list))
 
 (defn learning-list []
-  (->> (empty-list)
+  (->> (new-list)
```

Comment:
Should we be assuming knowledge of threading macros at this point?

Reply:
I don't think we should. I'll change the example and my reference to threading macros in the hints.